### PR TITLE
JS: Merge SSTI query into js/code-injection

### PR DESCRIPTION
**Pull request by asgerf**
This merges the Server-Side Template Injection from the experimental folder into the mainline `js/code-injection` query, essentially by adding all the sinks to that query.

There are a few reasons to merge the queries rather than have a separate SSTI query:
- They both lead to code execution.
- In both cases, we want to step through HTML sanitizers, as such sanitizers are ineffective.
- A combined query generally query runs faster than two separate queries, when the only difference is the set of sinks.

I made sure the alert message differentiates template injection, to emphasize that the template may contain code.

### CODE REVIEW

**Comment by asgerf** [comment]
[Evaluation](https://github.com/github/asgerf-dist-compare-reports/tree/js/template-sinks-in-code-injection_1606896231263) (internal link) looks mostly fine. The webpack outlier seems to be a fluke.

**Review by erik-krogh — COMMENTED** [review]
:+1: from me. 

You just need to update another `.expected` file.

**Comment by mchammer01** [comment]
Do you need a review from me on this @asgerf? (Sorry, just asking as I lately reviewed a JS query help PR as requested, because my name is in the codeowners file, but a review from me wasn't in fact needed).

**Comment by asgerf** [comment]
@mchammer01 yes please, I added a few paragraphs to the qhelp that could do with a review 👍

File: javascript/ql/src/Security/CWE-094/CodeInjection.qhelp
Comment (mchammer01) [inline javascript/ql/src/Security/CWE-094/CodeInjection.qhelp:62]: This link 404's for me. Could you check that the URL is correct?

File: javascript/ql/src/Security/CWE-094/CodeInjection.qhelp
Comment (mchammer01) [inline javascript/ql/src/Security/CWE-094/CodeInjection.qhelp:35]: There are 3 examples split across 2 separate "Example" sections. It might be best to put them all under one single "Examples" heading. What do you think?

Additionally, it would be nice to add a solution or workaround to the new example you added, or is it obvious to folks?

**Review by mchammer01 — APPROVED** [review]
@asgerf - LGTM ✨ 
There is a link in the References section that leads to a 404, and I've made a couple of minor comments (feel free to ignore the latter if you think they are not justified).
Approving this now so I am not blocking you (I have the day off today). Hope this helps 🙂

File: javascript/ql/src/Security/CWE-094/CodeInjection.qhelp
Comment (asgerf) [reply javascript/ql/src/Security/CWE-094/CodeInjection.qhelp:35]: 👍  Merged the two sections.

The solution to the new example is in the second paragraph below.

File: javascript/ql/src/Security/CWE-094/CodeInjection.qhelp
Comment (asgerf) [reply javascript/ql/src/Security/CWE-094/CodeInjection.qhelp:62]: Removed the link. Whatever it was pointing to I can't find it now.

**Review by mchammer01 — APPROVED** [review]
@asgerf - that's for addressing my comments, it's good to go from my perspective 👍🏻

**Review by erik-krogh — APPROVED** [review]

File: javascript/ql/src/Security/CWE-094/CodeInjection.qhelp
Comment (intrigus-lgtm) [reply javascript/ql/src/Security/CWE-094/CodeInjection.qhelp]: @asgerf were you looking for this:
https://owasp.org/www-project-web-security-testing-guide/stable/4-Web_Application_Security_Testing/07-Input_Validation_Testing/18-Testing_for_Server-side_Template_Injection.html
?

**Pull request merged**
